src/deluge/storage/flash_storage.cpp

```diff
@@ -197,6 +202,8 @@ bool gridEmptyPadsCreateRec;
 bool gridAllowGreenSelection;
 GridDefaultActiveMode defaultGridActiveMode;
 
+SliceModeDefault defaultSliceMode;
```

Comment:
this should be SampleRepeatMode defaultSliceMode

Reply:
aha! thx